src/lib/lightning/clightning/clightningService.ts

```diff
@@ -75,6 +75,7 @@ class CLightningService implements LightningService {
             localBalance: this.toSats(c.msatoshiToUs),
             remoteBalance: this.toSats(c.msatoshiTotal - c.msatoshiToUs),
             status,
+            isPrivate: c.private,
```

Comment:
Notice that I named the field `isPrivate` instead of a simpler `private` because `private` is a reserved keyword in JavaScript so I wanted to avoid it.

Reply:
Thanks for the rationale 👍